bika/lims/content/sampletype.py

```diff
@@ -106,6 +121,47 @@
             visibile=False,
         )
     ),
+    RecordsField(
+        'AdmittedStickerTemplates',
+        subfields=(
+            'admitted',
+            'small_default',
```

Comment:
Why `small_default` and `large_default` instead of `small` and `large`?

Since you use these strings in other places from the same class, better to define two global variable `STICKER_SMALL = 'small_default'` and `STICKER_LARGE = 'large_default'` and use them everywhere.

Reply:
`small_default` and `large_default` instead of `small` and `large` because from my point of view those are more descriptives names. As an example the description for `small_default` "Default small sticker"